platform/osx/os_osx.mm

```diff
@@ -3268,7 +3320,7 @@ void _update_keyboard_layouts() {
 	[GodotApplication sharedApplication];
 
 	// In case we are unbundled, make us a proper UI application
-	[NSApp setActivationPolicy:NSApplicationActivationPolicyRegular];
+	[NSApp setActivationPolicy:(is_no_window_mode_enabled() ? NSApplicationActivationPolicyAccessory : NSApplicationActivationPolicyRegular)];
```

Comment:
This won't work, `OS` is created before command line parsing, `is_no_window_mode_enabled()` will be always true.

Reply:
Fixed (but please check it's OK). Will come in the next push.